src/uct/ib/base/ib_md.c

```diff
@@ -1411,6 +1451,58 @@ uct_ib_md_parse_subnet_prefix(const char *subnet_prefix_str,
     return UCS_OK;
 }
 
+static double uct_ib_md_read_pci_bw(struct ibv_device *ib_device)
+{
+    const char pci_data_path_fmt[]   = "/sys/class/infiniband/%s/device/%s";
```

Comment:
could be const char *, to avoid stack usage

Reply:
hmmm, I expected that all const values are out of stack... but ok, fixed